openmdao/test_suite/test_examples/beam_optimization/test_beam_optimization.py

```diff
@@ -37,6 +43,105 @@ def test(self):
 
         print(prob['inputs_comp.h'])
 
+    def test_multipoint(self):
+        import numpy as np
+
+        from openmdao.api import Problem, ScipyOptimizeDriver
+
+        from openmdao.test_suite.test_examples.beam_optimization.multipoint_beam_group import MultipointBeamGroup
+
+        E = 1.
+        L = 1.
+        b = 0.1
+        volume = 0.01
+
+        num_cp = 5
+        num_elements = 50
+        num_load_cases = 2
+
+        prob = Problem(model=MultipointBeamGroup(E=E, L=L, b=b, volume=volume,
+                                                 num_elements=num_elements, num_cp=num_cp,
+                                                 num_load_cases=num_load_cases))
+
+        prob.driver = ScipyOptimizeDriver()
+        prob.driver.options['optimizer'] = 'SLSQP'
+        prob.driver.options['tol'] = 1e-9
+        prob.driver.options['disp'] = True
+
+        prob.setup()
+
+        prob.run_driver()
+
+        print(prob['interp.h'])
```

Comment:
shouldn't this be an assert?

Reply:
I can probably switch all the beam examples over to use asserts.